### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by Nenglish7**
- Consider coveralls for better code coverage.

### CODE REVIEW

**Comment by Nenglish7** [comment]
EasyDb class is big so ill put that in a separate pr.

File: composer.json [hunk 1]
```diff
@@ -1,64 +1,57 @@
...
-            "ParagonIE\\EasyDB\\Tests\\": "tests"
+            "ParagonIE\\EasyDB\\Tests\\": "./tests"
         }
     },
     "require": {
         "ext-pdo": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5",
-        "spatie/7to5": "^1.0",
```
Comment (paragonie-scott) [inline composer.json]: Can I ask why you got rid of 7to5?

Comment (Zegnat) [reply composer.json]: The update to PHPUnit also means you will not be able to run the tests on PHP 5.6 anymore (even after conversion with 7to5). PHPUnit 5 was the last to have PHP 5.6 compatibility.

Comment (inisik) [reply composer.json]: Maybe use `^5|^6|^7` so if a higher php version is used use phpunit 7 or phpunit 6. And if all fails use phpunit 5.

Comment (inisik) [reply composer.json]: But sooner or later you are gonna have to drop phpunit 5 as it is no longer supported and php > 5.6 is not supported also.

Comment (paragonie-scott) [reply composer.json]: Right. We'll continue to support PHP 5 through its support lifecycle, then we'll stop providing noncommercial support for any PHP 5 versions of our libraries.

**Comment by k00ni** [comment]
Could anyone tell me, what is the status here?

**Comment by slayerulan** [comment]
Any update regarding this PR?

**Comment by paragonie-security** [comment]
This PR diverged significantly from the development branch. We're going to close this and focus our efforts on cleaning up the docblocks before the next minor version release.

**Pull request closed without merging**